internal/lattice/lattice.go

```diff
@@ -302,6 +316,12 @@ func (la *Lattice) Dot(w io.Writer) {
 			}
 		}
 	}
+	return edges
+}
+
+// Dot outputs the lattice in the graphviz dot format.
+func (la *Lattice) Dot(w io.Writer) {
```

Comment:
__[gometalinter]__
* [gocyclo] cyclomatic complexity 12 of function (*Lattice).Dot() is high (> 10)